pkg/raft/testdata/lagging_commit.txt

```diff
@@ -134,7 +134,7 @@ process-ready 1
 Ready MustSync=false:
 Messages:
 1->2 MsgHeartbeat Term:1 Log:0/0 Commit:13
-1->3 MsgHeartbeat Term:1 Log:0/0 Commit:11
+1->3 MsgHeartbeat Term:1 Log:0/0 Commit:13
 
 # Since the heartbeat message does not bump the follower's commit index, it will
```

Comment:
This comment now needs updating, since we fixed the issue that it describes: we can see that the entries are now committed earlier by one round of communication.

Reply:
Done.